Sources/Client/Model/Channel/Channel+Requests.swift

```diff
@@ -43,7 +43,7 @@ public extension Channel {
     ///   - options: an additional channel options, e.g. `.presence`
     ///   - completion: a completion block with `ChannelResponse`.
     @discardableResult
-    func watch(options: QueryOptions = [], _ completion: @escaping Client.Completion<ChannelResponse>) -> Cancellable {
+    func watch(options: QueryOptions = [], _ completion: @escaping Client.Completion<ChannelResponse> = { _ in }) -> Cancellable {
```

Comment:
Let's make it optional instead of `{ _ in }` default

Reply:
We have a lot of like that completion blocks. Probably we need to make it all optional. What do you think?

Reply:
It makes sense to have them all optional, but we can refactor them as we go, no need to do them all now

Reply:
Let's do all together later.

Reply:
Since we're touching this now, I suggest we refactor this now. We refactor as we go, less work later on.

Reply:
For all of them?

Reply:
I think we should be consistent. For this one specific will look weird.

Reply:
It's not this one specific, we already have optional completion in some places
eg
```swift
public func set(user: User, token: Token, _ completion: Client.Completion<UserConnection>? = nil)
```

Reply:
It's easy to do that now for all of them.

Reply:
So, the set user is different it's not a request.
I found it's not so easy to make it optional because our base request works with non-optional completion block, that's why everywhere empty completion block.